language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -582,6 +582,8 @@ resetSandbox :: Sandbox-> IO ()
 resetSandbox sandbox = runWithSandbox sandbox $ do
     lid <- getLedgerIdentity
     Ledger.reset lid
+    lid2 <- getLedgerIdentity
+    unless (lid /= lid2) $ fail "resetSandbox: reset did not change the ledger-id"
```

Comment:
I always have a hard time wrapping my head around double negation. I suggest to use `when (lid1 == lid2) $ ...`.